src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -439,6 +439,16 @@ public void testSetters() {
         }
     }
 
+    @Test
+    public void testSetSeverity() throws Exception {
```

Comment:
Please add `@SuppressForbidden`
In separate PRs we do huge effort to avoid usage of `verify`
But this test has no choice  , and ok to use this method

Reply:
if I add this, the test file will exceed the limit of 2000 lines, it will start giving error. What should I do to solve this error?

Reply:
error log:
```
[ERROR] [checkstyle] [ERROR] C:\Open Source\checkstyle\src\test\java\com\puppycrawl\tools\checkstyle\CheckerTest.java:1: File length is 2,002 lines (max allowed is 2,000). [FileLength]
```

Reply:
Please add it to suppression, it might be not first for suppression .

Search keyword `FileLength` in config folder

Reply:
Done